.eslintrc

```diff
@@ -20,6 +20,7 @@
         "ignoreUrls": true
       }
     ],
+    "@typescript-eslint/no-unused-vars": ["error", { "argsIgnorePattern": "^_" }],
```

Comment:
Maybe controversial but I want to implement the interfaces correctly in the tests, but sometimes I legit don't care that the values are consumed, only that there's no escape hatch to bolt in a broken implementation in the public API